src/widgets/splits/Split.cpp

```diff
@@ -1014,7 +1007,12 @@ void Split::showViewerList()
                 if (currentCategory.empty())
                     continue;
 
-                chattersList->addItem(labelList.at(i));
+                auto label = formatListItemText(
+                    QString("%1 (%2)")
+                        .arg(labels.at(i))
+                        .arg(localizeNumbers(currentCategory.size())));
```

Comment:
If there are multiple substitutions of the same type, [`multi-arg overload`](https://github.com/KDE/clazy/blob/master/docs/checks/README-qstring-arg.md) should be used instead of making unnecessary temporary heap allocations with multiple `.arg(QString string)` calls:
```suggestion
                        .arg(labels.at(i), localizeNumbers(currentCategory.size())))
```